.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

I would suggest this

```suggestion
  - package-ecosystem: "gomod"
    directory: "/"
    open-pull-requests-limit: 10  # avoid spam, if no one reacts
    schedule:
      interval: "weekly"
  - package-ecosystem: "github-actions"
    directory: "/"
    open-pull-requests-limit: 10  # avoid spam, if no one reacts
    schedule:
      interval: "weekly"
```

---

sure, that works for me as well. The one I sent is the default one I drop into every repo I work with (if it hasn't one yet). :D